homeassistant/components/input_select.py

```diff
@@ -139,3 +173,19 @@ def select_option(self, option):
             return
         self._current_option = option
         self.update_ha_state()
+
+    def select_next(self):
```

Comment:
sure, the internal function could do something like ``` newIndex = ( currentIndex + offsetIndex) % len(options) ``` (not tested) instead of the ifs.